xrpl/clients/client.py

```diff
@@ -3,21 +3,24 @@
 
 from abc import ABC, abstractmethod
 
+from xrpl.models.parameters import ClientParameters
 from xrpl.models.requests.request import Request
 from xrpl.models.response import Response
 
 
 class Client(ABC):
     """Interface for all network clients to follow."""
 
-    def __init__(self: Client, url: str) -> None:
+    def __init__(self: Client, url: str, client_parameters: ClientParameters) -> None:
```

Comment:
This is much more of a JS paradigm than a Python paradigm - for Python you should just have an additional optional param. 
This would also be considered a breaking change and shouldn't be introduced at this time, but if you switch to an optional param then it wouldn't be a breaking change.

Reply:
I'm also not convinced that this should be something in the client - I feel like this should be a constant that exists in `xrpl.transaction` with an optional (defaulted to True) check that determines if the fee is too high.

Reply:
Thanks for your useful suggestion. I removed `client_parameters` from the Client and implemented that fee check, in `xrpl.transaction`. The check is an optional field (default to True).
I feel it's much cleaner and easier to read now, let me know what you think @mvadari